src/towncrier/_builder.py

```diff
@@ -276,11 +275,11 @@ def get_indent(text):
         # If bullets are not assumed and we wrap, the subsequent
         # indentation depends on whether or not this is a bullet point.
         # (it is probably usually best to disable wrapping in that case)
-        if all_bullets or text[:2] == u"- " or text[:2] == u"* ":
-            return u"  "
+        if all_bullets or text[:2] == "- " or text[:2] == "* ":
+            return "  "
         elif text[:3] == "#. ":
-            return u"   "
-        return u""
+            return "   "
```

Comment:
@hynek it looks like this is a missing code coverage.

I expect that this is very easy to cover, by updating the format used by an existing test.

Reply:
Could it be that there was a subtle bug or something working by accident before? 🤔

Reply:
Nope. Just missing test.

Here is an example of an updated test that will provide full cover for this feature

```diff
diff --git a/src/towncrier/test/test_build.py b/src/towncrier/test/test_build.py
index 66a1123..671f2e2 100644
--- a/src/towncrier/test/test_build.py
+++ b/src/towncrier/test/test_build.py
@@ -622,6 +622,9 @@ class TestCli(TestCase):
     def test_bullet_points_false(self):
         """
         When all_bullets is false, subsequent lines are not indented.
+
+        The automatic ticket number inserted by towcier will allign with the
+        manual bullet.
         """
         runner = CliRunner()
 
@@ -634,7 +637,24 @@ class TestCli(TestCase):
                 )
             os.mkdir("newsfragments")
             with open("newsfragments/123.feature", "w") as f:
-                f.write("wow!\n~~~~\n\nAdds levitation.")
+                f.write(
+                    "wow!\n"
+                    "~~~~\n"
+                    "\n"
+                    "No indentation at all."
+                    )
+            with open("newsfragments/124.bugfix", "w") as f:
+                f.write(
+                    "#. Numbered bullet list."
+                    )
+            with open("newsfragments/125.removal", "w") as f:
+                f.write(
+                    "- Hyphen based bullet list."
+                    )
+            with open("newsfragments/126.doc", "w") as f:
+                f.write(
+                    "* Asterisk based bullet list."
+                    )
 
             result = runner.invoke(
                 _main,
@@ -666,8 +686,29 @@ class TestCli(TestCase):
             wow!
             ~~~~
 
-            Adds levitation.
+            No indentation at all.
             (#123)
+
+
+            Bugfixes
+            --------
+
+            #. Numbered bullet list.
+               (#124)
+
+
+            Improved Documentation
+            ----------------------
+
+            * Asterisk based bullet list.
+              (#126)
+
+
+            Deprecations and Removals
+            -------------------------
+
+            - Hyphen based bullet list.
+              (#125)
             """
             ).lstrip(),
         )
```

Reply:
But how come we seem to have lost it now? I'm just on my phone ATM so I can't check but the old mixture of u and non-u strings looks sketchy? 🤔